### DISCUSSION TITLE
Using Cloudflare for Micros

### DISCUSSION THREAD

**Discussion by lthon09** [deta/cloud-docs#241] [General] [answered]
Hi! I'm asking whether it's possible to use Cloudflare for Deta Micros, since if I recall correctly, Cloudflare requires the base domain (without the subdomain) and that'd apply for all the existing Micros I think? Thanks!

**Comment by abdelhai** [comment]
hi, i'm not sure how to interpret your question.
you can use cloudflare dns (not proxy, make sure to disable the "orange cloud") without any issues with any micro.
we support root domains and sub-domains.

**Comment by lthon09** [comment]
i mean, to register my micro for cloudflare, id have to provide the root domain instead of my subdomain only, would this work? thanks!

**Reply by abdelhai** [reply]
my apology – i'm still now quite sure what you are asking.
i suggest trying out your plan and report any issue you might face.

you can find everything needed to work with domains/subdomains can be found on this page: https://docs.deta.sh/docs/micros/custom_domains

**Comment by lthon09** [comment, marked as answer]
ah, im not using cloudflare for a custom domain, but for some extensions and anti ddos. thanks!

**Reply by abdelhai** [reply]
oh, then this won't work as we don't support this behavior yet.
@aavshr is this something we could support easily?
